studio/components/ValueOrderedArrayOfObjectsInput.tsx

```diff
@@ -0,0 +1,28 @@
+import { ArrayOfObjectsInputProps, ObjectItem } from "sanity";
+
+interface OrderedArrayOfObjectsInputProps extends ArrayOfObjectsInputProps {
```

Comment:
bit heavy to read this 🥲 What about orderedArrayInput or orderedObjectArray

Reply:
This was to emphasize that it's an extension of the original `ArrayOfObjectsInputProps`. It should actually be even longer:`ValueOrderedArrayOfObjectsInputProps` 🙃 

We need `...Input...` and `...Props` to follow convention and differentiate with the component. So either `OrderedArrayInputProps` or `OrderedObjectArrayInputProps`. The former does not convey that this only covers arrays of objects (compared to primitives), and the latter is barely shorter with an inconsistent naming.

I prefer giving descriptive and fully qualified names by default, and then shorten them if they are used in a lot of places.